### ISSUE TITLE
Copyright protection

### ISSUE THREAD

**Issue by ForNeVeR** [codingteam/codingteam.org.ru#69]
_(Branching the discussion from https://github.com/codingteam/codingteam.org.ru/pull/68.)_

Alright, I've performed a small historical research of the situation around copyright and now I think I have all the pieces of the puzzle.

Basically, there're two widely recognized intellectual property conventions: [Berne Convention](http://www.wipo.int/treaties/en/text.jsp?file_id=283698) and [Universal Copyright Convention (UCC)](http://portal.unesco.org/en/ev.php-URL_ID=15381&URL_DO=DO_TOPIC&URL_SECTION=201.html). These two covers almost all of Earth countries (interestingly, there's exactly one country I know of that doesn't recognize Berne Convention but recognizes UCC, namely Cambogia).

Berne Convention doesn't require anything from us to be "registered" as the authors of our content; it automatically starts working at the moment of content creation. It's good, it protects our rights, and it is actually already enough. Although, to provide small additional protection of our rights in Cambogia, communicate the copyright owner name in a more clear way and to fill empty place in footers of our sites, we **opt-in** to follow the UCC that states (Article III, item 1):

> […] if from the time of the first publication all the copies of the work published with the authority of the author or other copyright proprietor bear the symbol © ac-companied by the name of the copyright proprietor and the year of first publication placed in such manner and location as to give reasonable notice of claim of copyright. 

That means that we should add a year of the "first" publication, our name and `©` sign somewhere in a notable place on our works. The mentioning of _first_ publication could be a subject of debate itself, but for now I think that the common practice is to count every change in a software piece as creating a **new** copy, and therefore count it as a fresh publication.

TL;DR: we don't actually need to keep the year of the first publication of the first version of our site. It should be enough to keep the latest year. And we're protected by the Berne Convention even in the case we aren't explicitly stating our copyright.

I suggest to remove the year of the first publication from the footer on the next update (probably next year?).

**Comment by Minoru** [comment]
Wow. And I haven't even finished reading the document [you mentioned in some earlier discussion](https://github.com/codingteam/loglist/pull/145#discussion_r53128051) ._.

Maybe I'll get my act together and find some supporting evidence for my decision in #68, but until then, I support your suggestion.

**Comment by Minoru** [comment]
#70 didn't close this somehow. Doing it manually.
